src/platform/site-wide/sass/overrides/uswds/focus.scss

```diff
@@ -0,0 +1,138 @@
+@mixin focus-gold-light-outline {
```

Comment:
It seems like these mixins could go in the design-system, since it's likely that anything built with it would have a need for them.